application/blueprints/datamanager/services/planning_data.py

```
@@ -32,26 +37,62 @@ def get_entity_count_for_organisation_and_dataset(
-        f"&dataset={dataset}"
-        f"&quality=authoritative"
-        f"&limit=500"
+    if total is None:
+        return _get_entities_sequentially(organisation_entity, dataset)
+
+    base_url = _org_entities_url(organisation_entity, dataset)
+    offsets = list(range(0, total, ENTITY_PAGE_SIZE))
+    urls = [f"{base_url}&offset={offset}" for offset in offsets]
+
+    entities = []
+    for offset, page in zip(offsets, fetch_pages_concurrently(urls)):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)